runtime/gc_base/StringTable.cpp

```diff
@@ -324,8 +324,8 @@ stringComparatorFn(struct J9AVLTree *tree, struct J9AVLTreeNode *leftNode, struc
 		if (isMetronome) {
 #if defined(J9VM_GC_REALTIME) 
 			/* Consider 'live' strings as 'higher' than their dead counterparts. */
-			BOOLEAN leftStringAlive = j9gc_objaccess_checkStringConstantLive(javaVM, left_s);
-			BOOLEAN rightStringAlive = j9gc_objaccess_checkStringConstantLive(javaVM, right_s);
+			BOOLEAN leftStringAlive = checkStringConstantLive(javaVM, left_s);
```

Comment:
isMetronome check is still around. I think we agreed it's redundant?

Reply:
We didn't discuss the check around these `checkStringConstantLive` apis . Removing the checks is not as clear, I would say its safer to leave them.

Reply:
After having another look, I can confirm that the check is redundant.  I'll go ahead and remove them